README.md

```diff
@@ -88,9 +88,41 @@ make html # create html documentation
 version-mp2-georchestra
 version-mp2
 ```
----
-**NOTE**
 
-When a stable branch is created, kindly update the `version.txt` with the new stable branch name and the same applies to Mapstore2 submodule
-
----
+## Release procedure
+
+All developments land in master branch first and releases will be created from a stable branch.
+As soon as a new major version is ready to be released (developments on master are completed and tested) a new stable branch will be created from master (YYYY.MM.xx, eg. 2022.02.xx).
+That naming convention has been agreed because typically a stable branch in mapstore2-georchestra refers to the corresponding one in MapStore.
+
+Each stable branch should always report inside the version.txt file the MapStore stable branch used as well as the corresponding release candidate stable version (eg. 2022.02.xx-RCX-geOrchestra).
+
+Given a stable branch in mapstore2-georchestra, further minor releases of the same series (eg. 2022.02) are delivered from it (eg. 2022.02.01).
+These should include the update to the latest stable branch of MapStore of the same series.
+
+All the releases are first delivered as release candidates, turned into release after acceptance.
+
+
+### Creating release candidate:
+- Given a stable branch in mapstore2-georchestra create a new pre-release and tag for it at
+[Release creation page](https://github.com/georchestra/mapstore2-georchestra/releases/new)
+using the following naming convention: `YYYY.MM.xx-RCX-geOrchestra` (eg. 2022.02.00-RC1-geOrchestra), 
+where `YYYY.MM` corresponds to the major version and `xx` is the number of the minor release of the same series; `RCX` is the number of release candidate.
+- Add exhaustive description of the pre-release and attach all the needed assets: 
+  - *.war file for mapstore2-georchestra build
+  - Link to the docker image in GeoSolutions Docker Hub
+  - Link to the corresponding release page in MapStore as soon as available
+  - Relevant configuration steps expected for the new version
+- If a new RC should be released simply keep updated the RC number (eg. 2022.02.00-RC2-geOrchestra)
+
+### Creating release
```

Comment:
- Given a RC already available and accepted
- From the current stable branch corresponding to the series version to be released, update the first line of the version.txt file reporting the version (tag name) that is going to be released using the following naming convention:  `YYYY.MM.xx-geOrchestra`, where `YYYY.MM` corresponds to the major version and `xx` is the number of the minor release of the same series (eg. 2022.02.00-geOrchestra)
- create a new release and tag for it at [Release creation page](https://github.com/georchestra/mapstore2-georchestra/releases/new) using the same name specified in _version.txt_
- Add exhaustive description of the release (including descriptions from all release candidates) and attach all the needed assets:
    - *.war file for mapstore2-georchestra build
    - Link to the docker image in GeoSolutions Docker Hub
    - Link to the corresponding release page in MapStore as soon as available
    - Relevant configuration steps expected for the new version
- Restore the first line of the _version.txt_ file to the previous version in the stable branch